mozilla/foundation.mozilla.org
# replace random with pseudorandom

Closes https://github.com/mozilla/foundation.mozilla.org/issues/2615

### how to test

1. add RANDOM_SEED=530910203 to your .env
2.  `rm network-api\db.sqlite3 && inv migrate && inv manage load_fake_data && inv runserver`
3. open `http://localhost:8000/en/privacynotincluded` in a tab,
4. rerun (1)
5. open the pni homepage in another tab
6. verify that the pages from different runs have identical product listing.


### what it should look like

using that specific random seed value, things should like this:

![screenshot_2019-02-07 privacy not included a buyer s guide for connected products](https://user-images.githubusercontent.com/177243/52434847-335a7180-2ac5-11e9-8d11-a96fff619fb5.jpg)
